src/core/Status/Status.stories.tsx

```
@@ -18,50 +16,69 @@ const withEmptySWRCache = (component: JSX.Element) => (
             },
           });
         }),
       },
     },
   },
   render: () =>
-    withEmptySWRCache(<Status statusUrl={statusUrl} refreshInterval={0} />),
+    withEmptySWRCache(<Status statusUrl={StatusUrl} refreshInterval={0} />),
 };
 
 export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)